### PR TITLE
Create, style, and populate the Tutorials section

### PR DESCRIPTION
**Pull request by yuliiiah**
## Describe the Change

This PR updates the Palette Docs to include a dedicated section for Tutorials.

## Changed Pages

💻 [Tutorials](https://deploy-preview-2689--docs-spectrocloud.netlify.app/tutorials/)

## Jira Tickets

🎫 [DOC-951](https://spectrocloud.atlassian.net/browse/DOC-951)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-951]: https://spectrocloud.atlassian.net/browse/DOC-951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docusaurus.config.js
Comment (karl-cardenas-coding) [inline docusaurus.config.js:127]:
```suggestion
```

Thanks for improving this logic. We can remove this now 😄

File: sidebars.js
Comment (karl-cardenas-coding) [inline sidebars.js:95]:
```suggestion
```

Can we remove this?

File: src/css/custom.scss
Comment (karl-cardenas-coding) [inline src/css/custom.scss:231]: @yuliiiah do we need this CSS override now that we are keeping tutorials outside of docs?

File: src/css/sidebars.scss
Comment (karl-cardenas-coding) [inline src/css/sidebars.scss:1]: Very Clean 😄

**Review by karl-cardenas-coding — COMMENTED** [review]
Digging the progress on this so far. Holding off approval until the we had the chance to address some of the questions

File: docs/docs-content/tutorials/tutorials.md
Comment (karl-cardenas-coding) [inline docs/docs-content/tutorials/tutorials.md:9]:
```suggestion
This documentation section provides hands-on tutorials you can complete in your environment to learn more about Palette and VerteX. Here, you will find tutorials covering the fundamentals of Palette to help you become a proficient power user and advanced topics that require more time and attention to comprehend. These tutorials will enable you to maximize Palette's ability to manage Kubernetes at scale. Check out the sections below to learn more about crucial Palette concepts.
```

@addetz @caroldelwing, I think we need to place a bit more emphasis on the value proposition this section of the docs will provide the user. I also removed the term Spectro Cloud, as it's a bit odd to refer to ourselves in the third person.

File: docs/docs-content/tutorials/tutorials.md
Comment (karl-cardenas-coding) [inline docs/docs-content/tutorials/tutorials.md:11]: We can keep this section for now, but what will we do when we have all the topics? Having a paragraph for each section may be a bit too much. I honestly think we can do without the sections below and just dive into the cards. We could use the section below to highlight new tutorials, specific topics, or important series, such as the getting started. I'll defer to you both on what approach you prefer.

File: docs/docs-content/tutorials/tutorials.md
Comment (karl-cardenas-coding) [inline docs/docs-content/tutorials/tutorials.md:31]:
```suggestion
visibility into the workloads active inside your cluster and cluster costs.
```
Ableism replacement

File: docs/docs-content/tutorials/tutorials.md
Comment (addetz) [reply docs/docs-content/tutorials/tutorials.md:11]: We wanted to give a little bit of context for now. When we have more sections, we could also consider making landing pages for each section. We felt it was a bit sparse to have only the introductions and the cards on the page.

File: src/css/custom.scss
Comment (yuliiiah) [reply src/css/custom.scss:230]: Yup, since the tutorials folder lives under docs-content! If we don't hide it from there, it will be visible in the sidebar 😅

File: package-lock.json
Comment (karl-cardenas-coding) [inline package-lock.json:1]: Hmm...not sure why we have a change here. Can you try to remove this file and then do an `npm ci` so that we get a clean lock file please?

File: sidebars.js
Comment (karl-cardenas-coding) [inline sidebars.js:61]: Thoughts on calling it Profiles & Packs?  The reason I suggest this is that it looks a bit odd with only Pack's related content.

**Review by karl-cardenas-coding — APPROVED** [review]
Approved 🚀

File: sidebars.js
Comment (yuliiiah) [reply sidebars.js:61]: I would say that, since the category is flat, our users will be able to see the packs-related tutorials immediately, so the current naming doesn't affect visibility 🤔 But if we want, there's no harm in renaming this section!

**Pull request merged**
